### PR TITLE
Add Tooltips for locking/unlocking buttons - cost tab

### PR DESCRIPTION
**Pull request by H-Trym**
[53207](https://statoil-proview.visualstudio.com/Roughtechs/_workitems/edit/53207/)
[53206](https://statoil-proview.visualstudio.com/Roughtechs/_workitems/edit/53206/)
[53205](https://statoil-proview.visualstudio.com/Roughtechs/_workitems/edit/53205/)

### CODE REVIEW

File: frontend/src/Components/Case/Components/ClickableLockIcon.tsx [hunk 1]
```diff
@@ -48,21 +58,34 @@ const LockIcon: React.FC<LockIconProps> = ({
...
-                color="#007079"
-                onClick={() => handleLockIconClick(clickedElement)}
-            />
+            <>
+
+                {isProsp ? (
+                    <Tooltip title="Show numbers from PROSP file">
+                        <Button variant="ghost_icon" color="secondary" onClick={() => handleLockIconClick(clickedElement)}>
+                            <ExcelHideIcon size={20} /></Button>
+                    </Tooltip>) : <Tooltip title="Show Calculated Numbers"><Button variant="ghost_icon" color="secondary" onClick={() => handleLockIconClick(clickedElement)}><CalculatorHideIcon size={20} /></Button></Tooltip>
```
Comment (taustad) [inline frontend/src/Components/Case/Components/ClickableLockIcon.tsx:73]: Put the second tooltip on a new line and make it more readable?
Same goes for line 86

File: frontend/src/Components/Case/Components/ClickableLockIcon.tsx [hunk 2]
```diff
@@ -48,21 +58,34 @@ const LockIcon: React.FC<LockIconProps> = ({
         }
     }
 
+    if (isProsp && !sharepointId) {
+        return <Tooltip title="To show numbers from PROSP, please add a PROPS file to the case."><Button variant="ghost_icon" color="secondary" disabled><DisabledExcelHideIcon size={20} /></Button></Tooltip>
```
Comment (taustad) [inline frontend/src/Components/Case/Components/ClickableLockIcon.tsx:62]: Change wording to what is used in the user story

Comment (H-Trym) [reply frontend/src/Components/Case/Components/ClickableLockIcon.tsx:62]: Fixed typo PROPS -> PROSP

File: frontend/src/Components/Case/Components/ClickableLockIcon.tsx (hunk 1 above)
Comment (H-Trym) [reply frontend/src/Components/Case/Components/ClickableLockIcon.tsx:73]: Switched from uppercase to lowercase letters,
Formatted the tooltips to be more readable.

**Review by taustad — APPROVED** [review]

**Pull request merged**